examples/plot_knockoff_aggregation.py

```diff
@@ -89,14 +90,15 @@ def single_run(
         X,
         y,
         estimator=LassoCV(
-            n_jobs=n_jobs,
+            n_jobs=1,
             verbose=0,
-            max_iter=1000,
+            max_iter=2000,
```

Comment:
Do we really need to use something else than the default ? I would like to keep the examples focussed on the main ideas.

Reply:
Now that we have reduced the problem size, do we still have that many warnings.
And I think that the warning are rather due to too low values in the alpha grid: increasing  `max_iter`  is not the right fix.